samples/Sentry.Samples.Maui/Sentry.Samples.Maui.csproj

```diff
@@ -7,6 +7,7 @@
       On Windows, we'll also build for Windows 10.
     -->
     <TargetFrameworks>net8.0-android</TargetFrameworks>
+    <RuntimeIdentifiers Condition="$(TargetFramework.Contains('-android'))">android-arm;android-arm64;android-x86;android-x64</RuntimeIdentifiers>
```

Comment:
Isn't that what this group of properties is doing?

https://github.com/getsentry/sentry-dotnet/blob/ed63749b2a7e6404aaed19ae12810863016a9926/samples/Sentry.Samples.Maui/Sentry.Samples.Maui.csproj#L65-L72

Reply:
Yes, my bad I didn't notice the line ` <RuntimeIdentifier Condition="'$(TargetPlatformIdentifier)' == 'android'      And '$(OSArchitecture)' == 'Arm64'">android-arm64</RuntimeIdentifier> ` for some reasons

However it doesn't work at all on my side

Reply:
Doesn't work?

Reply:
not on windows at least

Reply:
Any context? Like error messages etc.?

Might also be worth trying with the version-5.0.0 branch since there are quite a few changes ([to support net9.0](https://github.com/getsentry/sentry-dotnet/pull/3699)) in that branch.

Reply:
>Any context? Like error messages etc.?

I'm getting ADB0020: Mono.AndroidTools.IncompatibleCpuAbiException: The package does not support the CPU architecture of this device.

>Might also be worth trying with the version-5.0.0 branch since there are quite a few changes (https://github.com/getsentry/sentry-dotnet/pull/3699) in that branch.

I'm gonna try that

Reply:
The only difference between what you're trying to do in this PR and what already happens in the Samples project is you're adding `android-arm` and `android-x86`... so possibly just need to tweak the existing code block to cater to whichever of those allows you to run the sample of the particular physical you're trying to deploy to. 

Would be good to keep this in the same styles as all the others.